code/upstream-integration-tests/src/androidTest/java/com/adobe/marketing/mobile/edge/integration/util/TestSetupHelper.kt

```diff
@@ -65,10 +66,16 @@ object TestSetupHelper {
      * @return The interact URL with location hint applied.
      */
     fun createInteractURL(locationHint: String?): String {
+        // Timeout is in milliseconds
+        val sharedState = TestHelper.getSharedStateFor(IntegrationTestConstants.ExtensionName.CONFIGURATION, 10_000)
+        val edgeDomain = sharedState?.get(IntegrationTestConstants.ConfigurationKey.EDGE_DOMAIN) as? String
+        if (edgeDomain.isNullOrEmpty()) {
+            fail("Edge domain could not be fetched from the configuration shared state.")
+        }
```

Comment:
The edge domain is optional so instead of failing could you set the default value?
https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/src/main/java/com/adobe/marketing/mobile/EdgeConstants.java#L145